src/oscar/apps/communication/notifications/views.py

```diff
@@ -69,8 +69,9 @@ def get_queryset(self):
             recipient=self.request.user)
 
 
-class UpdateView(BulkEditMixin, generic.RedirectView):
+class UpdateView(BulkEditMixin, generic.View):
     model = Notification
+    http_method_names = ['post']
     actions = ('archive', 'delete')
     checkbox_object_name = 'notification'
     permanent = False
```

Comment:
Please remove this line (`permanent = False`) - it's also a hangover from the redirect view.